Peoplesoft/main.tf

```diff
@@ -0,0 +1,538 @@
+        },
+                   {
+            source_port_ranges =  "*" 
+            source_address_prefix = "*"              
+            destination_port_ranges = "9200" 
+            destination_address_prefix = "${local.subnetPrefixes["elasticsearch"]}"    # ob to Elastic Servers            
+        }
+    ]
+    elasticsearch_sr_inbound = [
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["webserver"]}"             
+            destination_port_ranges = "9200"
+            destination_address_prefix = "*"             
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["bastion"]}"                
+            destination_port_ranges = "22" 
+            destination_address_prefix = "*"             
+        }
+    ]
+
+    elasticsearch_sr_outbound = [
+                {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["application"]}"  # ob to Application Servers               
+            destination_port_ranges = "9033-9039" 
+            destination_address_prefix = "*"             # Need to support ASG
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["webserver"]}"  # ob to WebServers            
+            destination_port_ranges = "8000" 
+            destination_address_prefix = "*"            
+        }
+    ]
+
+        toolsclient_sr_inbound = [
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["application"]}"             
+            destination_port_ranges = "5985-5986"
+            destination_address_prefix = "*"             
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["bastion"]}"                
+            destination_port_ranges = "22" 
+            destination_address_prefix = "*"             
+        }
+    ]
+
+    toolsclient_sr_outbound = [
+
+    ]
+    
+    database_sr_inbound = [
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["application"]}"                 
+            destination_port_ranges =  "1521" 
+            destination_address_prefix = "*"             
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["bastion"]}"  # input from the Load Balancer only.            
+            destination_port_ranges =  "22" 
+            destination_address_prefix = "*"                
+        }
+    ]
+    database_sr_outbound = [
+    ]
+
+    ##########################################
+    ## VMs in these subnets will need Availability sets.
+    ##########################################
+  #  needAVSets = [ "presentation", "application" ]
+}
+
```

Comment:
Need Inbound and outbound for identity. 

-	Port 22 inbound only from bastion host (for maintenance support) [Over private IP]
-	Port 80 & 443 inbound and outbound: TCP protocol only [For HTTP and HTTPS connections]
-	Outbound 1521 to the DB subnet [Over private IP]

Reply:
Done.